index.php

```diff
@@ -1,1249 +1,1300 @@
+<?php
+/*
+Copyright (C) 2014-2015 Universitätsbibliothek Mannheim
+See file LICENSE for license details.
+
+Authors: Alexander Wagner, Stefan Weil
+
+References:
+
+File upload (general)
+
+* http://www.php.net/manual/en/features.file-upload.post-method.php
+
+File upload with dropzone
+
+* http://www.dropzonejs.com/
+* http://www.startutorial.com/articles/view/how-to-build-a-file-upload-form-using-dropzonejs-and-php
+* http://maxoffsky.com/code-blog/howto-ajax-multiple-file-upload-in-laravel/
+
+Websockets:
+
+* https://en.wikipedia.org/wiki/Server-sent_events
+* https://developer.mozilla.org/en-US/docs/WebSockets/Writing_WebSocket_client_applications
+* http://code.google.com/p/phpwebsocket/
+* http://dharman.eu/?menu=phpWebSocketsTutorial
+
+Keyboard input
+
+* http://jsfiddle.net/angusgrant/E3tE6/
+* http://stackoverflow.com/questions/3181648/how-can-i-handle-arrowkeys-and-greater-than-in-a-javascript-function-which
+* http://stackoverflow.com/questions/5597060/detecting-arrow-key-presses-in-javascript
+* http://www.quirksmode.org/js/keys.html
+
+Key symbols
+
+* http://www.tcl.tk/man/tcl8.4/TkCmd/keysyms.htm
+
+* wmctrl, suckless-tools (lsw, sprop, wmname, ...)
+
+* display.im6, evince
+
+Authorization
+
+* http://aktuell.de.selfhtml.org/artikel/php/loginsystem/
+
+Overlays
+
+* http://answers.oreilly.com/topic/1823-adding-a-page-overlay-in-javascript/
+
+*/
+
+    session_start();
+    if (isset($_REQUEST['monitor'])) {
+        $monitor = $_REQUEST['monitor'];
+        $_SESSION['monitor'] = $monitor;
+    } elseif (!isset($_SESSION['monitor'])) {
+        $_SESSION['monitor'] = '???';
+    }
+    $_SESSION['referer'] = 'index.php';
+    require('auth.php');
+
+    // Connect to database and get configuration constants.
+    require_once('DBConnector.class.php');
+    $dbcon = new DBConnector();
+
+    // Support localisation.
+    require_once('i12n.php');
+
+    $user = false;
+    if (isset($_SESSION['username'])) {
+        # PHP session based authorization.
+        $username = $_SESSION['username'];
+        $address = $_SESSION['address'];
+        $user = "$username@$address";
+    } elseif (isset($_SERVER['PHP_AUTH_USER'])) {
+        # .htaccess basic authorization.
+        $user = $_SERVER['PHP_AUTH_USER'];
+    }
+
+    $winvnc = CONFIG_START_URL . "/theme/" . CONFIG_THEME . "/winvnc-palma.exe";
+?>
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
+       "http://www.w3.org/TR/html4/strict.dtd">
+
+<html>
+
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>PalMA</title>
+
+<link rel="icon" href="theme/<?=CONFIG_THEME?>/favicon.ico" type="image/x-icon">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.min.css">
+<link rel="stylesheet" href="pure-min.css">
+<link rel="stylesheet" href="palma.css" type="text/css">
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>

```

Comment:
This new line adds a dependency on jquery. Do we need it? If yes, we should use a local copy of jquery.min.js (which is already available on the PalMA stations) and update the documentation.